### PR TITLE
fix #77506 digi24.ro

### LINKED ISSUE #77506: www.digi24.ro
### Issue URL (Anti Adblock Script)
 [https://www.digi24.ro/live/digi24](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.digi24.ro%2Flive%2Fdigi24)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polish Anti Adblock Filters, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by wout-codes**
#77506
Fixes broken video player too.

### CODE REVIEW

File: EnglishFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -883,6 +883,8 @@ webcartop.jp#@#.single-ads
 @@||oasjs.kataweb.it/adsetup_pcmp.js$domain=kataweb.it
 ! broken self-promo fix
 @@||auchan.ru/pokupki/media/banners/*.jpg$~third-party
+! https://github.com/AdguardTeam/AdguardFilters/issues/77506
+@@||pubads.g.doubleclick.net/ssai/event$domain=digi24.ro
```
Comment (zloyden) [inline EnglishFilter/sections/whitelist.txt:887]: Remove it from this section.

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -8241,6 +8241,8 @@ interia.pl#%#document.cookie = "lltsg=1";
 !---------------------------------------------------------------------
 !------------------------------ Romanian -----------------------------
 !---------------------------------------------------------------------
+! https://github.com/AdguardTeam/AdguardFilters/issues/77506
+@@||imasdk.googleapis.com/js/sdkloader/ima3_dai.js$important,domain=digi24.ro
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:8245]: I don't need this rule on my end with extension/app.
Add here `@@||pubads.g.doubleclick.net/ssai/event/*/master.m3u8$domain=digi24.ro`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/whitelist.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/whitelist.txt:887]: Hi.

File: EnglishFilter/sections/foreign.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:8245]: I need, otherwsise adguard gets detected. Pardon me, for the delay. I am currently working on a big project.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt:8245]: Ok, thanks.

**Review by zloyden — APPROVED** [review]

**Pull request merged**